src/common/utils.js

```diff
@@ -35,12 +35,13 @@ function toRippledAmount(amount: Amount): RippledAmount {
   };
 }
 
-const FINDSNAKE = /([a-zA-Z]_[a-zA-Z])/g;
 function convertKeysFromSnakeCaseToCamelCase(obj: any): any {
   if (typeof obj === 'object') {
     let newKey;
     return _.reduce(obj, (result, value, key) => {
       newKey = key;
+      // taking this out of function leads to error in PhantomJS
```

Comment:
It's a bit concerning because they should have the same semantics or else we might not be testing properly.